Code/Editor/AnimationContext.cpp

```diff
@@ -53,16 +53,6 @@ class CMovieCallback
     {
         // Only switch camera when in Play mode.
         GUID camObjId = GUID_NULL;
-        if (Params.cameraEntityId.IsValid())
-        {
-            // Find owner editor entity.
-            CEntityObject* pEditorEntity = CEntityObject::FindFromEntityId(Params.cameraEntityId);
-            if (pEditorEntity)
-            {
-                camObjId = pEditorEntity->GetId();
-            }
-        }
-
         // Switch camera in active rendering view.
         if (GetIEditor()->GetViewManager())
         {
```

Comment:
How do we confirm that this is no longer having an effect? If we don't need the camera object id, and it is always null, can the rest of the function be removed?

Reply:
Sounds like it, the call for  the camera object is always null

Reply:
the other call from viewManager for CameraObjectId checks for null and its always null so it will set the value is the registry for radians. I think i can safety remove the logic from the viewmanager. 

![image](https://github.com/o3de/o3de/assets/854359/98eac20f-7612-4f58-8ed4-c3b69575ab4a)

Reply:
seems like this is done to avoid setting the value in the registry when the animation is playing or is this a different path for the fov to change?